lib/chai/core/assertions.js

```diff
@@ -1633,12 +1643,14 @@ Assertion.addMethod('within', function (start, finish, msg) {
   if (doLength && objType !== 'map' && objType !== 'set') {
     new Assertion(obj, flagMsg, ssfi, true).to.have.property('length');
   }
+  
+  const isNumeric = x => ['Number', 'BigInt'].includes(_.type(x))
```

Comment:
Can we extract this to somewhere reusable? Seems to be repeated a few times

Reply:
Sure! I didn't want to add this to the global scope via `utils` because it adds another way to check if a value is numeric.